src/socket.rs

```diff
@@ -56,12 +56,11 @@ use crate::{MaybeUninitSlice, RecvFlags};
 /// use socket2::{Socket, Domain, Type};
 ///
 /// // create a TCP listener bound to two addresses
-/// let socket = Socket::new(Domain::IPV4, Type::STREAM, None)?;
+/// let socket = Socket::new(Domain::IPV6, Type::STREAM, None)?;
 ///
 /// let address: SocketAddr = "[::1]:12345".parse().unwrap();
 /// let address = address.into();
 /// socket.bind(&address)?;
-/// socket.bind(&address)?;
```

Comment:
I think this line shouldn't be removed as this example is for creating a TCP listener bound to two addresses, as the comment above says.